### ISSUE TITLE
alarm sometimes doesn't fire up at specified time when in monitoring without movement

### ISSUE THREAD

**Issue by sopos** [SeaPea/GentleWake#11]
I looked at the code and found out that when starting monitoring, the weak up event is also set to the desired time for sure. But actually sometimes it does not fire up until some movement is detected after the desired weak time.

**Comment by SeaPea** [comment]
Do you mean the alarm never activates or that it does not activate when you are moving around enough to trigger the alarm?

I have never had the alarm fail to activate, but sometimes the Pebble OS stops sending accelerometer data to apps, requiring a restart of the watch.

**Comment by sopos** [comment]
No. I will describe this morning. I have alarm set to 7:05. I had Pebble on my arm. I was sleeping deeply without moving so accelerometer was not triggered. I woke up approximately at 7:25, while no alarm was fired up yet. Then I looked at the watch to check the time and after that the alarm fired up as it normally would during monitoring.
The issue is that it did not fire up at 7:05 at all or I was sleeping that hard so I was not woken up, but that's less probable.
It would be good to have debug log to check what happened. Are you able to implement loging? I haven't noticed that in the code.

**Comment by SeaPea** [comment]
On the original Pebbles the app has reached the absolute limits of memory so I literally cannot add any more functionality except for the newer Pebbles, which I was trying to avoid. 

So you're saying that the alarm activated after you checked the time on your watch? 
Did you press any buttons beforehand?
Did you travel across a timezone the previous day? 
Do you have the system standby mode turned on?
